cmake/Corrosion.cmake

```diff
@@ -1397,3 +1397,74 @@ function(corrosion_install)
         message(FATAL_ERROR "install(EXPORT ...) not yet implemented")
     endif()
 endfunction()
+
+# Create a library target that will run cxxbridge on the given files on a crate
+# that is assumed to contain a module with #[cxx::bridge]
+# Call with corrosion_cxxbridge(<target> CRATE <crate> PATH <tomlPath> FILES [<filePath>, ...])
+# <target> is the name of the target created that will host the cxx interface
+# CRATE <crate> is the target name of the specific target generated by corrosion_import_crate
+# PATH <tomlPath> is the relative path from where this function is called to Cargo.toml of <crate>
+# FILES [<filePath, ...] is the relative paths to files with #[cxx::bridge]. These are relative to <tomlPath>
+# 
+# Example: corrosion_cxxbridge(myCxxTarget CRATE someCrate PATH someCratePath FILES src/lib.rs)
+function(corrosion_cxxbridge target)
```

Comment:
I was trying to keep it in line with cmake in general, like add_library or add_executable. But this part is definitely up to you and I will do that if you think that is the correct way!

Regarding the path, it makes sense to keep it the same, but I was wondering if there was a way we could skip it entirely, as in we already have to select a specific imported target by corrosion and it should already know the path of the Cargo.toml. Any way we could omit it entirely perhaps? If it requires too much changes, its maybe a breaking change for later.

Reply:
> Thinking about it again, I guess it does make sense for the main target to be the C++ target, since that is the target being modified (via target_sources, target_include_directories etc.). Maybe you just rename it to cxx_target?

Yup, can do!

> So for the time being I would suggest to leave it as is and I would add the necessary information in a follow-up PR.

And yeah, I think that is a reasonable approach, not to change too much.